test/testdata/namer/dynamic_constant.rb

```diff
@@ -3,6 +3,6 @@
 class A
   def self.foo
     X = 10
-  # ^^^^^^ error: Dynamic constant assignment
+#   ^ error: Parse Error: dynamic constant assignment
```

Comment:
ditto https://github.com/sorbet/sorbet/pull/1670/files#r322410162

Can we get rid of the `Parse Error:` prefix?